app/backend/src/couchers/models.py

```diff
@@ -141,10 +141,7 @@ class User(Base):
     my_travels = Column(String, nullable=True)  # CommonMark without images
     things_i_like = Column(String, nullable=True)  # CommonMark without images
     about_place = Column(String, nullable=True)  # CommonMark without images
-    # TODO: array types once we go postgres
-    languages = Column(String, nullable=True)
-    countries_visited = Column(String, nullable=True)
-    countries_lived = Column(String, nullable=True)
+    avatar_filename = Column(String, nullable=True)
```

Comment:
This shouldn't be here. I think it's from a messed up merge

Reply:
avatar_filename?

Reply:
Yeah

Reply:
Oops. Didn't notice it.